pkg/reconciler/identity/resources/iam_policy_bindings_test.go

```diff
@@ -45,9 +45,10 @@ var (
 	}}
 )
 
+// TODO The iam policy binding should be mocked in order to do UT, issue
```

Comment:
just saw this! thanks, you can add the same thing to the previous comment, plus the link to the issue. 
Here I think you can add it as well... thanks!